scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/RatioBasedEstimator.scala

```diff
@@ -48,6 +48,9 @@ abstract class RatioBasedEstimator extends ReducerEstimator {
     val threshold = RatioBasedEstimator.getInputRatioThreshold(conf)
 
     historyService.fetchHistory(info, maxHistory) match {
+      case Success(Nil) =>
```

Comment:
does fetchHistory return a List or a Seq? I'm wondering if Success(Nil) will still match say, Success(Vector()) if the underlying Seq type ever changes?